docs/sql-reference/sql-statements/data-definition/CREATE TABLE.md

```diff
@@ -237,48 +243,61 @@ Please use specified key columns and specified value ranges for partitioning.
+PARTITION BY RANGE ( <partition_column1> [, <partition_column2>, ... ] )
 (
-    PARTITION partition_name1 VALUES [("k1-lower1", "k2-lower1", "k3-lower1",...), ("k1-upper1", "k2-upper1", "k3-upper1", ...)),
-    PARTITION partition_name2 VALUES [("k1-lower1-2", "k2-lower1-2", ...), ("k1-upper1-2", MAXVALUE, )),
-    "k3-upper1-2", ...
+    PARTITION <partition_name1> VALUES [( "<lower_bound_for_partition_column1>" [ , "<lower_bound_for_partition_column2>", ... ] ), ( "<upper_bound_for_partition_column1?" [ , "<upper_bound_for_partition_column2>", ... ] ) ) 
+    [,
+    PARTITION <partition_name2> VALUES [( "<lower_bound_for_partition_column1>" [ , "<lower_bound_for_partition_column2>", ... ] ), ( "<upper_bound_for_partition_column1>" [ , "<upper_bound_for_partition_column2>", ... ] ) ) 
+    , ...]
 )
 ```
 
 Note:
 
 - Fixed Range is more flexible than LESS THAN. You can customize the left and right partitions.
 - Fixed Range is the same as LESS THAN in the other aspects.
+- When only one column is specified as the partitioning column, you can set `MAXVALUE` as the upper bound for the partition column in the latest partition.
 
-#### Create partitions in bulk
+#### Create multiple partitions at a time
 
 Syntax
 
-```Plain%20Text
-PARTITION BY RANGE (datekey) (
-    START ("2021-01-01") END ("2021-01-04") EVERY (INTERVAL 1 day)
-)
-```
+- If the partitioning column is of an date type.
+
+    ```sql
+    PARTITION BY RANGE (<partitioning_column>) (
+        START ("<start_date>") END ("<end_date>") EVERY (INTERVAL <N> <time_unit>)
+    )
+    ```
+
+- If the partitioning column is of an integer type.
+
+    ```sql
+    PARTITION BY RANGE (<partitioning_column>) (
+        START ("<start integer>") END ("<end integer>") EVERY (<partitioning_granularity>)
+    )
+    ```
 
 Description
 
-You can specify the value for `START` and `END` and the expression in `EVERY` to create partitions in bulk .
+You can specify the start and end value in `START()` and `END()` and the time unit or partitioning granunity in `EVERY()` to create multiple partitions at a time.
 
-- If `datekey` supports DATE and INTEGER data type, the data type of `START`, `END`, and `EVERY` must be the same as the data type of `datekey`.
-- If `datekey` only supports DATE data type, you need to use the `INTERVAL` keyword to specify the date interval. You can specify the date interval by day, week, month, or year. The naming conventions of partitions are the same as those for dynamic partitions.
+- The partitioning column can be of a date or integer type, and the data type of values in `START()` and `END()`, and `EVERY()` must be the same as the data type of partitioning column.
+- The partitioning column is of a data type, you need to use the `INTERVAL` keyword to specify the date interval. You can specify the time unit as hour (since v3.0), day, week, month, or year. The naming conventions of partitions are the same as those for dynamic partitions.
```

Comment:
a data type --- 这里应该是 date。If the partitioning column is of the DATE type, you need to use the `INTERVAL` keyword to specify the time interval. You can specify the time unit as hour (since v3.0), day, week, month, or year.